apps/app/atoms/userState.ts

```diff
@@ -0,0 +1,53 @@
+import { useEffect } from 'react'
+import { atom, useRecoilState } from 'recoil'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+// Types
+import { UserType } from '../types/UserType'
+
+/* --- Atoms ----------------------------------------------------------------------------------- */
+
+const userState = atom<null | UserType>({
+  key: 'user',
+  default: null,
+})
+
+/* --- useUserState() -------------------------------------------------------------------------- */
+
+const useUserState = () => {
+  // State
+  const [user, setUser] = useRecoilState(userState)
```

Comment:
in this case the useRecoilState is abstracted in a useUserState hook, but I assume the normal WOW with recoil is that you use recoil directly in your components? or do you see an advantage to always wrap it?